README.md

```diff
@@ -212,6 +212,24 @@ This is used to compare average CO2 emissions: if the user gives consent, her mo
 Then, she can compare herself with the average emissions of all the participating users.
 All data sent to the DACC is anonymized, and only aggregated values under a certain threshold can be queried.
 
+### Develop with the DACC
+
+To develop locally with the DACC, you need to:
```

Comment:
On peut peut-être indiquer au début quelque chose du genre : "avant toute chose vous devez avoir le token d'accès à `cc.cozycloud.dacc.dev_v2`

Reply:
Ok, I added a precision about that